src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -62,11 +62,12 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
+        $errorIo = $io->getErrorStyle();
```

Comment:
@xabbuh The framework has a conflict for `symfony/console: <3.3` so it should be fine, right?